package.json

```diff
@@ -26,15 +27,14 @@
   "devDependencies": {
     "@types/lodash": "^4.14.161",
     "@types/mathjs": "^6.0.11",
+    "esbuild": "^0.12.12",
+    "esbuild-serve": "^1.0.1",
```

Comment:
I was checking this package and it has only 13 stars on Github, I usually avoid packages with little engagement like this one. But it seems that esbuild already has an internal functionality to serve applications, [see](https://esbuild.github.io/api/#serve). If it fits for our case, using only esbuild is preferable.